getting-started/git.md

```diff
@@ -1,87 +0,0 @@
----
```

Comment:
just out of curiosity, we don't want this article?

Reply:
No, its going to get refactored and probably moved.